src/platform/errors/index.ts

```diff
@@ -81,6 +87,7 @@ function getCallSite(frame: stackTrace.StackFrame) {
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 type Constructor<T> = { new (...args: any[]): T };
+
```

Comment:
@DonJayamanne I saw your comment and I agree! I have undone the changes to isErrorType 🙂

Reply:
On the Azure SDK for JS, we used to only check errors based on `error.name`. I believe there was an issue with instanceof. It might have been when possibly referencing to different copies of the same class, which is something possible in Node.js (specially when two dependencies pull in different versions of an underlying common dependency). I don't think this (whatever might have been the root issue that stuck with me) is a problem in this repository! Also: The fewer changes, the better.